src/main/scala/org/zalando/spearheads/innkeeper/routes/DeleteDeletedRoutes.scala

```diff
@@ -44,7 +44,7 @@ class DeleteDeletedRoutes @Inject() (
       dateTimeParameter(deletedBefore) match {
         case Some(dateTime) =>
           team(authenticatedUser, token, requestDescription) { team =>
-            (isAdminTeam(team, requestDescription)(teamService) | hasOneOfTheScopes(authenticatedUser, requestDescription, scopes.ADMIN)) {
+            hasAdminAuthorization(authenticatedUser, team, requestDescription, scopes)(teamService) {
               removeDeleteBeforeRoutes(authenticatedUser, dateTime, requestDescription)
             } ~ reject(InnkeeperAuthorizationFailedRejection(requestDescription))
```

Comment:
this part can be removed, isn't it? `~ reject(InnkeeperAuthorizationFailedRejection(requestDescription))`